src/test/java/ui/ConfigProperties.java

```diff
@@ -0,0 +1,31 @@
+package ui;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConfigProperties {
+    protected static Properties properties;
+    protected static FileInputStream fileInputStream;
+
+    static {
+        try {
+            fileInputStream = new FileInputStream("src/test/resources/config.properties");
+            properties = new Properties();
+            properties.load(fileInputStream);
+        } catch (IOException e) {
+            e.printStackTrace();
+        } finally {
```

Comment:
use `try-with-resources` construction to avoid such boilerplate.
```
try(final InputStream is = new FileInputStream("...")){
  //Do whatever you want inside with InputStream
  //The stream will be closed gracefully after the block.
}
```